### DISCUSSION TITLE
Use different objects to mark overlapping object outlines

### DISCUSSION THREAD

**Discussion by samytichadou** [bnpr/Malt#582] [Q&A]
Hi i would like to mark (through lines or rim light) overlapping outlines between 2 separate objects.

Here is a screenshot : 
![image](https://github.com/user-attachments/assets/6c14f112-e667-4ca9-ad2e-1c9936167353)
My goal is to get malt to generate the yellow lines automatically.
I am struggling with getting this kind of effect, i am trying to use object ID with pre pass, but i'm still unclear with the how (new user here, sorry if dumb question :) )

Does anyone out here has an idea ? cheers !

**Comment by pragma37** [comment]
You could set the `Custom ID A` to (0,0,0,0) (the same as the background) for the red shape and to something else for the blue shape.
Then if you use a Line Width node to the red shape where all Widths are 0 except for the second channel of `ID Boundary Width` that should get you the result you want.
